WG_JS_CORE.md

```diff
@@ -97,34 +96,92 @@ Improving the docs, specs and having great tutorials and examples goes a long wa
   - Return values are documented, including object properties and types
 - Onboarding documentation exists to explain domain concepts and provide tutorials/examples for achieving common tasks
 
-### `M (P0)`: Revamped APIs 🔄 🤝 🧠
+### `M (P0)`: Revamped APIs 📦 🤝
 
 Revamping the APIs will make them easier to use and simpler to understand. Using modern syntax will make JS IPFS appealing to a wider segment of the JS community, aiding adoption and contributions. It also gives us scope to make performance optimisations and stability/debuggability improvements.
 
 - Async/await is used throughout the codebase and callback APIs are dropped
-- Support ArrayBuffer as an alternative to Buffer input argument
+- ArrayBuffer is supported as an alternative to Buffer input argument
 - Streaming APIs switch to using async iterators/iterables
 - The files API is revamped:
     - Files functions moved to the root namespace
     - Non-MFS and MFS functions are merged
     - A revamped RESTful HTTP API exists
 - Old APIs are deprecated and removed (e.g. object, block)
```

Comment:
Maybe add 'Support the DAG API over http', otherwise we can't really remove the object API.

Reply:
I'd like to see the object API being removed, but I think the block API still makes sense even if we have a proper DAG API.